src/style.css

```diff
@@ -0,0 +1,92 @@
+body {
+  font-family: "Montserrat", sans-serif;
+  overflow: hidden;
+  margin: -21px 0 0 0;
+}
+
+.container {
+  display: flex;
+}
+
+.bg-one {
+  background-color: #6befa3;
+  height: 105vh;
+  width: 40vw;
+  position: relative;
+  z-index: 0;
+  overflow: hidden;
+}
+
+.bg-one::before {
+  content: "";
+  position: absolute;
+  background-color: #efefef;
+  height: 150%;
+  width: 100%;
+  right: 0;
+  margin-right: calc(-150% / 2);
+  top: 50%;
+  margin-top: calc(-1450px / 2);
+  border-radius: 50% 0% 0% 50%;
+  z-index: -1;
+}
+
+.bg-two {
+  content: "";
+  height: 105vh;
+  width: 60vw;
+  background-color: #efefef;
+}
+
+.footer {
+  bottom: 97px;
+  right: 188px;
+  position: fixed;
+  text-align: center;
+}
+
+.footer p {
+  font-size: 16px;
+  color: black;
+  font-weight: 400;
+}
+
+@media (min-device-width: 0px) and (max-device-width: 750px) {
```

Comment:
I strongly recommend you to make your CSS mobile first, whenever possible.

Reply:
Agreed. Moreover, this media query seems a bit redundant. `max-device-width: 750px` implicit includes `min-device-width: 0px`.

As @marmitoTH mentioned, following a mobile-first approach is strongly advised, but the general idea should have your styling in a linear way, i.e.:

```
/* styling without media queries (usually mobile) */

@media (min-device-width: 768px) {
    /* styling for tablet */
}

@media (min-device-width: 1024px) {
    /* styling for desktop */
}

```

Or maybe the other way around. Without queries for desktop, then tablet and mobile. 

Either way, this will help you reusing styles on multiple screen sizes, as styles will be passed down and overwritten/unset according to your needs.

If needed, you can also go a bit further and create more breakpoints to encompass larger tablets, larger mobile devices, widescreen desktops, etc.

**Obs:** you can also consider using properties such as `min-width` and `max-width` when you want your queries to trigger on different screen resolutions only. `(min/max)-device` will be useful especially if you want to build different layouts for each screen orientation (portrait/landscape) on the same device resolution.